### PR TITLE
Add SMTP configuration

### LINKED ISSUE #110: Add SMTP settings
## Purpose

Pixel cannot send emails for now. It could be at least useful to email `settings.ADMINS` on 500 errors.

## Proposal

- [x] Configure `settings.Base.ADMINS`
- [x] Configure `settings.Development.EMAIL_BACKEND` setting to use the [console-backend](https://docs.djangoproject.com/en/1.11/topics/email/#console-backend)
- [x] Configure `Staging` & `Production` to use environment variable for `EMAIL_*` settings
- [x] Add environment variables to production|staging `.env`

### PR DESCRIPTION
**Pull request by willdurand**
Fix #110

---

This PR adds a new `ADMINS` and email settings.

### CODE REVIEW

File: pixel/settings.py [hunk 1]
```diff
@@ -24,6 +24,8 @@ class Base(Configuration):
 
     DEBUG = False
 
+    ADMINS = [('Julien Maupetit', 'julien@tailordev.fr'), ]
```
Comment (jmaupetit) [inline pixel/settings.py]: I think a generic email address like: `pixel@candihub.eu` that redirects to @gaellelelandais @biorosetics and myself (during the development phase) must be preferred.

WDYT @gaellelelandais & @biorosetics?

File: pixel/settings.py [hunk 2]
```diff
@@ -152,15 +154,41 @@ class Development(Base):
...
+    EMAIL_HOST=smtp.example.org
+    EMAIL_PORT=587
+    EMAIL_HOST_USER=babar
+    EMAIL_HOST_PASSWORD=KingOfTheElephants
+    """
 
     ALLOWED_HOSTS = ['staging.pixel.candihub.eu', ]
 
+    EMAIL_HOST = values.Value(
+        '', environ_name='EMAIL_HOST', environ_prefix=None)
```
Comment (jmaupetit) [inline pixel/settings.py]: Syntactically speaking, I prefer when the closing bracket position matches the block it closes, _e.g._:

```python
EMAIL_HOST = values.Value(
    '', environ_name='EMAIL_HOST', environ_prefix=None
)
```

File: pixel/settings.py [hunk 3]
```diff
@@ -152,15 +154,41 @@ class Development(Base):
...
+        '', environ_name='EMAIL_HOST_PASSWORD', environ_prefix=None)
+    EMAIL_USE_TLS = True
+    EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+    EMAIL_SUBJECT_PREFIX = '[Pixel/staging] '
+    DEFAULT_FROM_EMAIL = "Pixel Admin <no-reply@pixel.candihub.eu>"
+    SERVER_EMAIL = DEFAULT_FROM_EMAIL
+
 
-class Production(Base):
+class Production(Staging):
```
Comment (jmaupetit) [inline pixel/settings.py]: 👍

File: pixel/settings.py (hunk 1 above)
Comment (willdurand) [reply pixel/settings.py]: :+1:

Comment (gaellelelandais) [reply pixel/settings.py]: I agree. A generic adress is a good idea.

**Pull request merged**